tfx/experimental/templates/taxi/e2e_tests/stub_kubeflow_e2e_test.py

```diff
@@ -0,0 +1,358 @@
+# Lint as: python3
+# Copyright 2020 Google LLC. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""E2E test using kubeflow orchestrator with stub executors for taxi template."""
+
+import datetime
+import os
+import subprocess
+import tarfile
+import urllib.request
+import time
+
+from absl import logging
+import kfp
+import tensorflow as tf
+from typing import Text
```

Comment:
This should go to the upper block because typing is standard python library.

https://google.github.io/styleguide/pyguide.html#313-imports-formatting